Assignment 3-JS/sass/style.scss

```diff
@@ -0,0 +1,235 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+html {
+  font-size: 80%;
+}
+.container-body {
+  width: 100%;
```

Comment:
Block level element takes default 100% width.

Reply:
Removed width of block element.